yt/utilities/lib/platform_dep.h

```diff
@@ -1,59 +1,14 @@
 #include <math.h>
 #ifdef MS_WIN32
 #include "malloc.h"
+/*
+note: the following implicitly sets a mininum VS version: conservative
+minumum is _MSC_VER >= 1928 (VS 2019, 16.8), but may work for VS 2015
```

Comment:
```suggestion
minimum is _MSC_VER >= 1928 (VS 2019, 16.8), but may work for VS 2015
```
will fix after tests run...